l10n/en/products/monitor/shared.ftl

```diff
@@ -14,6 +14,9 @@ monitor-shared-sub-nav-all-breaches = All breaches
 
 ## Footer
 
+monitor-take-back-control = Take back control of your data
+monitor-breach-data-provided-by = Breach data provided by <strong>Have I Been Pwned</strong>
```

Comment:
Nit: If these are to become shared strings then the new strings here should have the naming prefix `monitor-shared-`. The landing page should be updated to use the new shared string IDs, and can use the old ones as a fallback?